pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -322,3 +324,11 @@ def _check_on_hpc_hooks(model: "pl.LightningModule") -> None:
             "Method `LightningModule.on_hpc_load` is deprecated in v1.6 and"
             " will be removed in v1.8. Please use `LightningModule.on_load_checkpoint` instead."
         )
+
+
+def _check_on_configure_sharded_model(trainer: "pl.Trainer", model: "pl.LightningModule") -> None:
```

Comment:
model is unused and doesn't need to be an argument to this function 
```suggestion
def _check_on_configure_sharded_model(trainer: "pl.Trainer") -> None:
```